CustomItems/Items/Scp5355.cs

```diff
@@ -0,0 +1,168 @@
+﻿
+namespace CustomItems.Items
+{
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using Exiled.API.Extensions;
+    using Exiled.API.Features;
+    using Exiled.CustomItems.API;
+    using Exiled.CustomItems.API.Features;
+    using Exiled.CustomItems.API.Spawn;
+    using Exiled.Events.EventArgs;
+    using Hints;
+    using MEC;
+    using UnityEngine;
+
+
+    public class Scp5355 : CustomWeapon
+    {
+        private readonly Dictionary<Player, Vector3> sizedPlayers = new Dictionary<Player, Vector3>();
+        private int mode = 1;
```

Comment:
You are giving the same variable to all SCP-5335, so one player with the weapon can change the mode for the other

Reply:
Do I need to use a Dictionary?

Reply:
Probably yeah, but I'm not sure how would it be easilly handled by CustomItems API so I would ask Amathor abotu that.

Reply:
@galaxy119

Reply:
CustomItem classes are managers, they handle all the logic for all the custom items of their type on the map and inside inventories.

It's probably best to track the mode settings for each player individually, using a Dictionary<Player, int> is probably the best way to accomplish this.